driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudTest.java

```diff
@@ -43,6 +43,22 @@ public static void doSkips(final String fileDescription, final String testDescri
             assumeFalse(testDescription.equals("Aggregate with $out includes read preference for 5.0+ server"));
             assumeFalse(testDescription.equals("Database-level aggregate with $out includes read preference for 5.0+ server"));
         }
+        if (fileDescription.equals("updateOne-sort")) {
+            assumeFalse(testDescription.equals("UpdateOne with sort option"));
```

Comment:
Let's add a second param for each of these, like `"Skipped until implementation of JAVA-XXXX"`, where XXXX is the issue number for adding sort to update/replace.